### DISCUSSION TITLE
Fatal: the remote end hung up unexpectedly Error

### DISCUSSION THREAD

**Discussion by JeekoCZ** [community/community#23736] [New to GitHub]
<p>Hello!</p>
<p>I’m trying to push 200,000+ files but every time I tried to push the files it always came out with an error. I’ve tried pushing about 3 times by now but it’s always the same error.</p>
<p>The log is <a href="https://pastebin.com/SkZcwnDg" rel="nofollow noopener">here</a></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeekocz">@JeekoCZ</a>, welcome to the community! Our <a href="https://help.github.com/en/github/managing-large-files/what-is-my-disk-quota#file-and-repository-size-limitations">File and repository size limitations page</a> has guidance about git repositories and mentions that:</p>
<pre><code class="lang-auto">We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended. 
Smaller repositories are faster to clone and easier to work with and maintain. 
Individual files in a repository are strictly limited to a 100 MB maximum size limit.
</code></pre>
<p>Are you able to check your repositories and confirm if any files exceed the 100MB per-file maximum size limit?</p>
<p>If it does, then you will need to either follow our <a href="https://help.github.com/en/github/managing-large-files/removing-files-from-a-repositorys-history">Removing files from a repository’s history</a> guide to delete the file, or use git’s <a href="https://help.github.com/en/github/managing-large-files">Large File Storage (LFS)</a> feature to track and store this (and any other large) files, before the repository can be pushed to <a href="http://GitHub.com">GitHub.com</a>.</p>

**Comment by JeekoCZ** [comment]
<p>Hello,</p>
<p>I did indeed exceed the limit.<br>
It’d be nice if instead of spitting out an error it instead told you what is wrong.</p>
<p>Anyways thank’s for the quick reply!</p>
